### ISSUE TITLE
Locally build the whole project is failing

### ISSUE THREAD

**Issue by zhfeng** [quarkiverse/quarkus-artemis#585]
I run with `./mnvw clean install -DskipTests` on my local machine. It is failing with
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-install-plugin:3.1.2:install (default-install) on project quarkus-integration-test-artemis-camel-jms-common: The packaging plugin for project quarkus-integration-test-artemis-camel-jms-common did not assign a main file to the project but it has attachments. Change packaging to 'pom'. -> [Help 1]
```

It seems related to the new version of `maven-install-plugin:3.1.2` see https://issues.apache.org/jira/browse/MINSTALL-151
There is a work around to add `-DallowIncompleteProjects=true`

Is there anybody who met this same issue?

**Comment by turing85** [comment]
This was already addressed in https://github.com/quarkiverse/quarkus-artemis/pull/586/commits/60965e8b345c1326bf5c25b41684c12b75af0bbb

**Comment by turing85** [comment]
The fix has been released with versions [`3.4.1`](https://github.com/quarkiverse/quarkus-artemis/releases/tag/3.4.1) and [`3.2.3`](https://github.com/quarkiverse/quarkus-artemis/releases/tag/3.2.3).
